src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,19 +1,43 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    print("A:", arrA, "B:", arrB)
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
     # TO-DO
-    
+
```

Comment:
another way to do this would be to have 2 variables, one looking at arrA and one looking at arrB and increment them instead of del aaA[0] you could increment arrA_point += 1.  Deleting from the beginning of the array is inefficent and is O(n) runtime because the array has to be re indexed.